transport/http/client.go

```diff
@@ -59,6 +61,13 @@ func SetClientBefore(before ...RequestFunc) ClientOption {
 	return func(c *Client) { c.before = before }
 }
 
+// SetClientAfter sets the ClientResponseFuncs applied to the incoming HTTP
+// request prior to it being decoded. This is useful for obtaining anything off
+// of the response and adding onto the context prior to decoding.
+func SetClientAfter(after ...ClientResponseFunc) ClientOption {
```

Comment:
Yeah, definitely. This is all set. I also renamed SetClientBefore in transport/grpc.Client to match what is in transport/grpc.Client. If it's not a problem, at a later point in time I'll update one of the examples (probably addsvc) to use Client.ClientAfter, but let me know if that's something that should be part of this PR.